cmd/notary/keys.go

```diff
@@ -322,11 +307,11 @@ func (k *keyCommander) keysRestore(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	ks, err := k.getKeyStores(config, true)
+	ks, err := k.getKeyStores(config, true, true)
```

Comment:
Note: this was originally on `keyRestore` :(  I must have commented on an out of date diff or something - got moved.